cmd/worker/internal/embeddings/repo/handler.go

```diff
@@ -252,16 +252,18 @@ func (r *revisionFetcher) Diff(ctx context.Context, oldCommit api.CommitID) (
 		}
 
 		switch f.Status {
-		case gitdomain.DeletedAMD:
+		case gitdomain.StatusDeleted:
 			// Deleted since "oldCommit"
 			toRemove = append(toRemove, f.Path)
-		case gitdomain.ModifiedAMD:
+		case gitdomain.StatusModified:
 			// Modified in "r.revision"
 			toRemove = append(toRemove, f.Path)
 			changedNew = append(changedNew, f.Path)
-		case gitdomain.AddedAMD:
+		case gitdomain.StatusAdded:
 			// Added in "r.revision"
 			changedNew = append(changedNew, f.Path)
+		default:
```

Comment:
Would it be more helpful for future readers to specifically enumerate the ones we deliberately ignore (e.x. gitdomain.StatusTypeChanged, ....

```suggestion
		case gitdomain.StatusTypeChanged:
			// Deliberately ignore StatusTypeChanged
		default:
			// Add a log or error message here for unexpected status types
```